ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -42,6 +52,32 @@ public void beforeEach() {
   @Test
   public void testContextLoads() {}
 
+  @Test
+  public void testSuccessfulRequest() throws Exception {
+    when(restTemplate.postForObject(
+            eq("https://localhost/transform"), any(HttpEntity.class), eq(TransformResponse.class)))
+        .thenReturn(new TransformResponse());
+
+    mvc.perform(
+            multipart("/ingest")
+                .file("file", "some-content".getBytes())
+                .param("fileSize", "10")
+                .param("fileName", "file")
+                .param("title", "qualityTitle")
+                .param("mimeType", "plain/text")
+                .header("Accept-Version", "1.2.1")
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.MULTIPART_FORM_DATA))
+        .andExpect(status().isOk());
+
+    verify(restTemplate, times(1))
+        .postForObject(
+            eq("https://localhost/transform"), any(HttpEntity.class), eq(TransformResponse.class));
+  }
+
+  @Test
+  public void testUnsuccessfulRequest() {}
```

Comment:
Is this empty in anticipation for the real test?

Reply:
Yes, I should have probably put a TODO there.